modules/openxr/doc_classes/OpenXRAPIExtension.xml

```diff
@@ -228,20 +228,23 @@
 			<param index="0" name="extension" type="OpenXRExtensionWrapper" />
 			<description>
 				Registers the given extension as a composition layer provider.
+				[b]Note:[/b] This cannot be called after the OpenXR session is running. However, it can be called in [method OpenXRExtensionWrapper._on_session_created].
```

Comment:
```suggestion
				[b]Note:[/b] This cannot be called once the OpenXR session is running. However, it can be called in [method OpenXRExtensionWrapper._on_session_created].
```
Or
```suggestion
				[b]Note:[/b] This cannot be called after the OpenXR session has started. However, it can be called in [method OpenXRExtensionWrapper._on_session_created].
```
Unless "after it has finished running" is meant, for all